shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java

```diff
@@ -118,7 +120,7 @@ public boolean skip(final ServerWebExchange exchange) {
                 .exchange()
                 .doOnError(e -> LOG.error(e.getMessage(), e))
                 .timeout(Duration.ofMillis(timeout))
-                .retryWhen(Retry.onlyIf(x -> x.exception() instanceof ConnectTimeoutException)
+                .retryWhen(Retry.anyOf(TimeoutException.class, ConnectTimeoutException.class, ReadTimeoutException.class)
```

Comment:
can add exception.class?

Reply:
> can add exception.class?

How about `Throwable`? If ok, I'll change to `reactor.retry.Retry#any`.